tekton/mario-bot/mario-github-comment.yaml

```diff
@@ -42,117 +42,154 @@ spec:
+          comment_params = dict(pic_alt=chosen_pic, pic_src=pic_url)
+            
+          if successful:
+            comment_template += (
+            'Here is the image you requested: '
+            '<a href="https://{imageurl}">built image</a>|'
             )
-
-            if successful:
-              chosen_pic = random.choice(ok_pics)
-            else:
-              chosen_pic = random.choice(failed_pics)
-            pic_url = "/".join([marios_pics_root, chosen_pic]) + '.png'
-            comment_params = dict(pic_alt=chosen_pic, pic_src=pic_url)
-
-            if successful:
-              comment_template += (
-                'Here is the image you requested: '
-                '<a href="https://{imageurl}">built image</a>|'
-              )
-              comment_params['imageurl'] = '$(resources.outputs.image.url)'
-            else:
-              comment_template += (
-                'Cloud not build the requested image. Please check the '
-              )
-
+            comment_params['imageurl'] = '$(params.targetImage)'
+          else:
             comment_template += (
-              '<a href="http://35.222.249.224/?buildid={buildid}&'
-              'namespace=mario">build logs</a>'
+            'Could not build the requested image. Please check the '
             )
-            comment_params['buildid'] = '$(tt.params.buildUUID)'
-
-            new_comment_path = "$(resources.outputs.pr.path)/comments/new.json"
-            comment_body = dict(body=comment_template.format(**comment_params))
-            with open(new_comment_path, "w") as comment:
-              json.dump(comment_body, comment)
-      resources:
-        inputs:
-          - name: source
-            resourceSpec:
-              type: git
-              params:
-              - name: revision
-                value: $(tt.params.gitRevision)
-              - name: url
-                value: $(tt.params.gitURL)
-          - name: pr
-            resourceSpec:
-              type: pullRequest
-              params:
-                - name: url
-                  value: $(tt.params.gitURL)/pull/$(tt.params.pullRequestID)
-              secrets:
-                - fieldName: authToken
-                  secretName: mario-github-token
-                  secretKey: GITHUB_TOKEN
-        outputs:
-          - name: image
-            resourceRef:
-              name: $(tt.params.targetImageResourceName)
-          - name: pr
-            resourceSpec:
-              type: pullRequest
-              params:
-                - name: url
-                  value: $(tt.params.gitURL)/pull/$(tt.params.pullRequestID)
-              secrets:
-                - fieldName: authToken
-                  secretName: mario-github-token
-                  secretKey: GITHUB_TOKEN
+            
+          comment_template += (
+            '<a href="http://35.222.249.224/?buildid={buildid}&'
+            'namespace=mario">build logs</a>'
+          )
```

Comment:
The logs application is not used anymore, we might want to update this with a link to the tekton dashboard: https://dashboard.dogfooding.tekton.dev/#/namespaces/mario/pipelineruns